datafusion/catalog-listing/src/helpers.rs

```diff
@@ -569,218 +580,6 @@ mod tests {
         assert_eq!(0, chunks.len());
     }
 
-    #[tokio::test]
-    async fn test_pruned_partition_list_empty() {
```

Comment:
Theses tests look like unit tests to me (they are are for `pruned_partition_list` which is defined in this module

I looked at `make_test_store_and_state` and it seems to me like it is fairly simple and could be copy/pasted into this `test` module to avoid a new dependency. 

Would you be ok with doing that? I can also push a commit to this PR if it is good with you

Reply:
I took the liberty of doing this in [53dbc2f](https://github.com/apache/datafusion/pull/14464/commits/53dbc2fb1b175cfaed4fe94f2331f8f2fcd71d1c)